MekHQ/src/mekhq/gui/dialog/DataLoadingDialog.java

```diff
@@ -212,13 +202,13 @@ public Void doInBackground() {
                     fis.close();
                 } catch (NullEntityException e) {
                     JOptionPane.showMessageDialog(null,
-                            "The following units could not be loaded by the campaign:\n"
-                                    + e.getError() + "\n\nPlease be sure to copy over any custom units"
-                                    + "before starting a new version of MekHQ.\nIf you believe the units"
-                                    + "listed are not customs, then try deleting the file data/mechfiles/units.cache"
-                                    + "and restarting MekHQ.\nIt is also possible that unit chassi"
-                                    + "and model names have changed across versions of MegaMek."
-                                    + "You can check this by\nopening up MegaMek and searching for the units."
+                            "The following units could not be loaded by the campaign:\n "
+                                    + e.getError() + "\n\nPlease be sure to copy over any custom units "
+                                    + "before starting a new version of MekHQ.\nIf you believe the units "
+                                    + "listed are not customs, then try deleting the file data/mechfiles/units.cache "
+                                    + "and restarting MekHQ.\nIt is also possible that unit chassi "
```

Comment:
```suggestion
                                    + "and restarting MekHQ.\nIt is also possible that unit chassis "
```

Reply:
Actually, chassi is proper English in this case. This is because Chassi is the plural of chassis, which is what makes the most sense for the dialog.